cmd/start/cmd.go

```diff
@@ -176,6 +166,23 @@ func enqueueNewBlocks(exportQueue types.HeightQueue, ctx *parser.Context) {
 	}
 }
 
+// mustGetLatestHeight will keep trying until it gets the latest height from RPC client, panic after 10 retries
+func mustGetLatestHeight(ctx *parser.Context) (int64, error) {
```

Comment:
Hmm how about we add maximum number of retries and if the number gets exceeded we skip that block and let the `daily_refetch` module refetch fix it later? Otherwise `mustGetLatestHeight` function might get stuck in loop forever